makePDF.py

```diff
@@ -75,7 +84,167 @@ def getOEMCP():
     return str(codepage)
 
 
+class LatextoolsBuildSelector(sublime_plugin.WindowCommand):
+
+	# on 3080+, the necessary features are built-in by default
+	if sublime.version() > '3079':
+		def run(self, select=False):
+			select = False if select not in (True, False) else select
+			self.window.run_command('build', {'select': select})

```

Comment:
For example write an EventListener

``` py
class SublimeVersionListener(sublime_plugin.EventListener):
    def on_query_context(self, view, key, operator, operand, match_all):
        if key != "_lt_st_build_version":
            return
        return sublime.version() <= '3079'
```
And add a context to the keybinding:
``` js
{ "keys": ["ctrl+b"],
  "context": [
    {"key": "selector", "operator": "equal", "operand": "text.tex.latex"}
    {"key": "_lt_st_build_version"}],
  "command": "latextools_build_selector"},
```

This avoids to call the command at all and we don't overwrite keybindings of other packages. I think this is a better solution than calling the other command from this one.